test/Results.test.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import Results from "../src/components/Results";
+
+describe('should render Results component', () => {
+  const element = mount(<Results items={10} />);
+  test('Snapshot test with default props', () => {
+    expect(element).toMatchSnapshot();
+  });
+});
+
+describe('should render Results component with 10 items', () => {
+  const element = mount(<Results items={10} />);
+  expect(element.prop('items')).toEqual(10);
```

Comment:
I'd suggest changing this test to check if value provided in `items` prop is properly rendered by component.

Checking if prop value is equal to provided value, tests if React works properly, not component.

Reply:
Done.